src/components/menu-surface/menu-surface.scss

```diff
@@ -2,6 +2,11 @@
 @use '@material/elevation';
 @use '@material/menu';
 
+/**
+* @prop --limel-menu-surface-display: defines whether the surface is treated as a block, flex or grid. It affects layout used to display its children. Defaults to `block`.
+* @prop --limel-menu-surface-flex-direction: defines the direction of menu-surface layout.
+*/
```

Comment:
Just in case someone else thinks, as I did, "isn't this a new feature?": no, because it's only documented on this private component, consumers won't know about them, and also, if we make sure we always define a value for these somewhere inside the limel-menu component, any value the consumer sets won't actually get here.